src/containers/Carousel.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <Swiper
+    class="h-full w-full"
+    :modules="[Autoplay, A11y, Navigation, Keyboard]"
+    :autoplay="{
+      disableOnInteraction: false,
+    }"
+    :free-mode="true"
+    :keyboard="{ enabled: true }"
+    :lazy-preload-prev-next="1"
+    :space-between="-70"
+    :slides-per-view="1"
+    :loop="true"
+    :loop-prevents-sliding="true"
+    @real-index-change="
+      (swiper) => {
+        currentSlideIndex = swiper.realIndex;
+      }
+    "
+  >
+    <SwiperSlide
+      class="flex h-full w-full items-center justify-center"
+      style="display: flex"
+      v-for="(recommendation, index) in recommendations"
+      :key="index"
+      @click="handleSlideClick(index)"
+    >
+      <div
+        class="relative flex items-center justify-center rounded-3xl shadow-light transition-all dark:shadow-dark"
+        :class="
+          isTargetIndex(currentSlideIndex, index)
+            ? 'z-10 h-[460px] w-[340px] cursor-pointer'
+            : 'my-[46px] h-[368px] w-[272px]'
+        "
+      >
+        <RecipeCard
+          :recipe="recommendation.recipe"
+          v-if="isTargetIndex(clickedSlideIndex, index)"
+        />
+        <img
+          :src="recommendation.cover.imageUrl"
+          :alt="recommendation.cover.title"
```

Comment:
최대한 방어로직을 적용해주는게 좋습니다.
```suggestion
          :src="recommendation.cover.imageUrl ?? "기본이미지URL" "
          :alt="recommendation.cover.title ?? "기본이미지Title" "
```